bdd/features/e2e/E2E-014-python.feature

```diff
@@ -0,0 +1,118 @@
+Feature: Test our shiny new Python runner
```

Comment:
We'll need to change the line eventually to... `Test our rusty old Python runner` ;)